target/linux/ath79/image/generic.mk

```diff
@@ -538,22 +538,23 @@ TARGET_DEVICES += glinet_gl-ar150
 define Device/glinet_gl-ar300m-common-nor
   ATH_SOC := qca9531
   DEVICE_VENDOR := GL.iNet
-  DEVICE_MODEL := GL-AR300M
```

Comment:
This change is introduced in the previous patch.
I think it would be better to already use the logic from this patch there.